ansible/library/parse_text_table.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/python
+from ansible.module_utils.basic import *
+
+DOCUMENTATION = '''
+---
+module:  parse_text_table
+version_added:  "2.0"
+author: Joe Lazaro (@joeslazaro)
+short_description: Parse a text table and convert it to a dict or list of dicts
+description: |
+    Expects a text table with with fixed-width columns and '-' 
+    characters as separators between the column headers and the data. It will
+    product a list of dicts corresponding to each row of data. It will also
```

Comment:
There's a typo here. The word 'product' should be 'produce'. Can you go ahead and fix that?

Reply:
fixed the typo per your request